### PR TITLE
VoxelSniper: The journey through the Fabric of time. Part two: Implementing the project

### PR DESCRIPTION
**Pull request by KevinDaGame**
## What type of PR is this? (check all applicable)

- [x] Feature
- [ ] Refactor
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Implement the fabric (empty) project
## QA Instructions, Screenshots, Recordings

https://www.youtube.com/watch?v=YPN0qhSyWy8

## Added/updated tests?

- [ ] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
  have not been included_
- [ ] I need help with writing tests
- [ ] Not needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: VoxelSniperFabric/build.gradle [hunk 1]
```diff
@@ -0,0 +1,90 @@
+plugins {
+    id 'fabric-loom'
+    id 'maven-publish'
```
Comment (Lennart99) [inline VoxelSniperFabric/build.gradle:3]: Do we need the maven-publish plugin?

**Review by Lennart99 — APPROVED** [review]
maybe it would be good to put this on a separate branch for now, similar to what we did with Forge?

File: VoxelSniperFabric/build.gradle (hunk 1 above)
Comment (KevinDaGame) [reply VoxelSniperFabric/build.gradle:3]: I don't think we do actually. I'll remove it

**Comment by KevinDaGame** [comment]
> maybe it would be good to put this on a separate branch for now, similar to what we did with Forge?

I disagree. I want to take a different approach than we did with forge. I found it very difficult to work on, because the branches deviated so much. In addition, keeping those seperate was needed because we did the major refactor. That shouldn't be necessary this time.

While our Fabric version is in alpha, we won't guarantee stability at all, so it's use at your own risk. We will however enforce master to compile, so our releases should stay stable.

**Comment by Lennart99** [comment]
> > maybe it would be good to put this on a separate branch for now, similar to what we did with Forge?
> 
> I disagree. I want to take a different approach than we did with forge. I found it very difficult to work on, because the branches deviated so much. In addition, keeping those seperate was needed because we did the major refactor. That shouldn't be necessary this time.
> 
> While our Fabric version is in alpha, we won't guarantee stability at all, so it's use at your own risk. We will however enforce master to compile, so our releases should stay stable.

Sure, both is fine I guess.

**Pull request merged**